src/helpers/numbers.js

```diff
@@ -15,6 +15,7 @@ export const numberFormat = (amount, currency = null, locale = userLocale, maxim
     const params = { style, minimumFractionDigits: maximumFractionDigits, maximumFractionDigits };
     if (currency) {
         params.currency = currency;
+        params.currencyDisplay = "narrowSymbol";
```

Comment:
This will make it show `$` instead of `US$` when your locale is like `en-IN` or `en-GB`